### PR TITLE
doctl formula 0.0.16

### PR DESCRIPTION
**Pull request by thebyrd**
https://github.com/digitalocean/doctl

### CODE REVIEW

**Comment by mattkanwisher** [comment]
This formula is pretty cool its similar to aws command line tools, but allows you to control computers and cloud services at DigitalOcean. +1

File: Library/Formula/doctl.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
+require "language/go"
+
+class Doctl < Formula
+  desc "command-line tool for DigitalOcean"
+  homepage "https://github.com/digitalocean/doctl"
+  url "https://github.com/digitalocean/doctl/archive/0.0.16.tar.gz"
+  sha256 "4f4805f36fd0d437331c25a183471419d10a680721f7f5a890b4109319d605ed"
+  head "https://github.com/digitalocean/doctl.git"
+
+  bottle do
```
Comment (DomT4) [inline Library/Formula/doctl.rb]: Please don't add bottle blocks. We'll do that automatically on pull.

**Comment by thebyrd** [comment]
@DomT4 Thanks! I knew there had to be an easier way to get those. I've removed them now.

**Comment by thebyrd** [comment]
ping @DomT4

**Comment by DomT4** [comment]
There was a downstream build issue. I've stuck it back in the queue, which is the hold-up here.

**Comment by thebyrd** [comment]
Thanks! is it an issue I can help resolve?

**Comment by DomT4** [comment]
Na. Local to our CI. It's had a pretty much constant build queue for the last 4/5 days and there have been some issues with that, especially with us bringing the El Cap CI online progressively across the same period.

**Comment by thebyrd** [comment]
Would DigitalOcean credit help speed things up?

**Comment by DomT4** [comment]
I'm afraid technology is more or less immune to bribery :wink:. The hold up here isn't human. Just bear with us whilst we get it sorted.

**Comment by thebyrd** [comment]
Ah, I more meant if you needed more servers for ci we'd be happy to provide
them.

**Comment by DomT4** [comment]
Ah! Apologies for the misunderstanding. I presumed it was light humour and responded in kind.

OS X is theoretically legally tricky to emulate due to Apple's EULA and Apple has been proactive in enforcing their rights there. I believe Travis gets around this by doing testing on licensed Apple hardware, which is the approach we take as well. We're exploring a few options going forwards as handling our own CI is endless endless maintenance.

**Comment by thebyrd** [comment]
@DomT4 ping

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/8aecf757b0be1324b5c1fbf463a505684b631e4e. Thank you for your contribution to Homebrew @thebyrd; we appreciate it! :smiley_cat:

**Pull request closed without merging**